Whiskey/Functions/Install-WhiskeyNuGetPackage.ps1

```diff
@@ -0,0 +1,55 @@
+function Install-WhiskeyNuGetPackage {
+
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [string]$Name,
+        # The name of the NuGet package to download. 
+        
+        [string]$Version,
+        # The version of the package to download. Must be a three part number, i.e. it must have a MAJOR, MINOR, and BUILD number.
+
+        [Parameter(Mandatory)]
+        [string]$DownloadRoot
+        # The root directory where the tools should be downloaded. The default is your build root.
+        #
+        # PowerShell modules are saved to `$DownloadRoot\Modules`.
+        #
+        # NuGet packages are saved to `$DownloadRoot\packages`.
+    )
+        if( -not $IsWindows )
+        {
+            Write-Error -Message ('Unable to install NuGet-based package {0} {1}: NuGet.exe is only supported on Windows.' -f $Name,$Version) -ErrorAction Stop
+            return
+        }
+        $nugetPath = Join-Path -Path $PSScriptRoot -ChildPath '..\bin\NuGet.exe' -Resolve

```

Comment:
Use [$whiskeyBinPath](https://github.com/webmd-health-services/Whiskey/blob/develop/Whiskey/Whiskey.psm1#L20) to get the path to Whiskey's `bin` directory.